content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -163,97 +169,100 @@ Partner
 Identification Type and VAT
 ***************************
 
-As part of the Chilean localization, the identification types defined by the SII
-are now available on the Partner form. This information is essential for most transactions.
+As part of the Chilean localization, the identification types defined by the :abbr:`SII (Servicio de
+Impuestos Internos)` are now available on the partner form. This information is essential for most
+transactions.
 
-.. image:: chile/Chile06.png
+.. image:: chile/identification-type-vat.png
    :align: center
-   :alt: Chilean identification types for partners.
+   :alt: Chilean identification types and VAT for partners.
 
-TaxpayerType
-************
+Taxpayer Type
+*************
 
-In Chile, the document type associated with customers and vendors transactions is defined
-based on the Taxpayer Type. This field should be defined in the partner form, when creating
-a customer is important you make sure this value is set:
+In Chile, the document type associated with customers and vendors transactions is defined based on
+the :guilabel:`Taxpayer Type`. This field should be defined in the partner form, when creating a
+customer is important you make sure this value is set:
```

Comment:
```suggestion
the :guilabel:`Taxpayer Type`. This field should be defined in the partner form, when creating a
customer in the Contacts application, is important you make sure this value is set:
```
Since contact creation is also reviewed in other sections of the Odoo standard documentation I think it should not be a detailed explanation, but added the part referring to the Contacts application